test/browser/run-test.sh

```diff
@@ -44,9 +44,31 @@ if [ "$ID" = "rhel" ]; then
     "
 fi
 
-if [ "$TEST_OS" = "centos-8-stream" ]; then
-    EXCLUDES="$EXCLUDES TestMachinesConsoles.testExternalConsole"
-fi
+# We only have one VM and tests should take at most one hour. So run those tests which exercise external API
+# (and thus are useful for reverse dependency testing and gating), and exclude those which test internal
+# functionality -- upstream CI covers that.
+EXCLUDES="$EXCLUDES
+          TestMachinesCreate.testDisabledCreate
+
+          TestMachinesConsoles.testExternalConsole
+          TestMachinesConsoles.testInlineConsole
+          TestMachinesConsoles.testInlineConsoleWithUrlRoot
+          TestMachinesConsoles.testSerialConsole
```

Comment:
While for other types of consoles, we are using a patternfly components, serial console is from cockpit's own `cockpit-components-terminal.jsx`, so technically it's a bit different. So maybe that one should be tested? Or is it enough that  `cockpit-components-terminal.jsx`is tested with packit tests in cockpit repo?

Reply:
It seems enough to run that upstream. JSX components don't change with e.g. a newer kernel or libvirt version in Fedora/RHEL, this is all tightly under our upstream control. So this seems safe to skip.